hawkular-alerts-engine/src/main/resources/org/hawkular/alerts/schema/updates/schema-1.4.0.groovy

```diff
@@ -0,0 +1,32 @@
+/*
+ * Copyright 2015-2016 Red Hat, Inc. and/or its affiliates
+ * and other contributors as indicated by the @author tags.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *    http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+setKeyspace keyspace
+
+schemaChange {
+  version '3.0'
+  author 'jshaughn'
+  tags '1.4.x'
+  cql "DROP TABLE alerts_statuses"
+}
+
+schemaChange {
+    version '3.0'
```

Comment:
Version on schema changes should be something sequencial (3.0, 3.1).
If not it might overwrite the internal log.
I.e. https://github.com/hawkular/hawkular-metrics/blob/master/core/schema/src/main/resources/org/hawkular/schema/updates/schema-0.21.0.groovy#L26
(5.1, 5.2, 5.3...)